src/content/smart-bulletins/032.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Smart Bulletin No. 032"
+intro: "Updates to Commercial Interchange Fee for Visa and Mastercard Products"
+number: "No. 032"
+order: 032
+businessline: "Purchase, Travel, VISA/MasterCard Fleet, and Integrated"
+published: "Dec 20, 2019"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| December 20, 2019 | Implementation of Smart Bulletin #32 |
+
+## Effective Date
+
+April 12, 2019 (VISA) and April 13, 2019 (MasterCard)
+
+## Business Line(s) Affected
+
+Purchase, Travel, VISA/MasterCard Fleet, and Integrated
+
+## Introduction
+
+VISA and MasterCard have increased the interchange fee, which is assessed for processing the transfer of funds between issuing banks and merchant banks. 
+
+## Summary
+
+1. In a charge card, consumer, or commercial transaction, the merchant’s bank pays the interchange fee to the issuing bank. The merchant’s bank passes along the cost of the interchange fee to the merchant and the merchant may factor in the cost of the association’s interchange fee in pricing for goods and services provided.
```

Comment:
In a charge card, consumer, or commercial transaction, the merchant’s bank pays the interchange fee to the issuing bank. The merchant’s bank passes along the cost of the interchange fee to the merchant, and the merchant may factor in the cost of the association’s interchange fee in pricing for goods and services provided.
The interchange fee varies and is influenced by factors, which may include but are not limited to the type of merchant, region of the world, type of transaction, payment solution used, and amount transacted.
The increase in the interchange fee may be nominal and will not impact services currently provided or negotiated refund rates currently in effect for the GSA SmartPay 3 Program.